desktop/core/requirements.txt

```diff
@@ -74,4 +74,4 @@ thrift==0.13.0
 thrift-sasl==0.4.2
 git+https://github.com/gethue/django-babel.git
 git+https://github.com/gethue/django-mako.git
-werkzeug==2.0.1  # Should remove it here and from devtools.mk
+werkzeug==2.0.1  # Should remove it here and from devtools.mk
```

Comment:
Could we not change that?

Reply:
I just hard reset my fork to upstream and its still saying its still a change, I have no idea?

Reply:
An easy way is to just manually add it back so